controllers/games.js

```diff
@@ -36,4 +37,21 @@ exports.getCommentsById = (request, response, next) => {
       response.status(404).send({ msg: "No such path found. Try again..." });
     }
   });
-};
\ No newline at end of file
+};
+
+exports.addCommentByReviewId = (request, response, next) => {
+  //console.log(request.params.review_id)
+  createsNewComment(request.body, request.params.review_id)
+    .then((comment) => {
+      if (comment) {
+      response.status(201).send({ comment });
+      } else {
+        response.status(404).send({ msg: "No such reviewID found. Try again..." });
```

Comment:
Could you handle this rejection in the model, so that it goes to the catch block instead? You can then pass it to the error handlers to deal with